campaignion_foundation.js

```diff
@@ -156,4 +156,35 @@ Drupal.behaviors.payment_slide.attach = function (context, settings) {
   });
 };
 
+/**
+ * Provide an event to act on form step changes.
+ *
+ * If no form step is yet known, assume we are initially loading a page with a
+ * webform.
+ *
+ * You can use this event to conditionally show/hide parts of a page.
+ *
+ * NB: We currently need to use native events to allow dispatching between
+ * different versions of jQuery (`.trigger()` and `.on()` communication
+ * between the Drupal's `$` and the the mo-foundation-base's `$` does not
+ * work).
+ */
+Drupal.behaviors.form_step_data = {};
+Drupal.behaviors.form_step_data.attach = function (context, settings) {
+  // current step is always a number > 0 if the settings information is correct
+  let currentStep = settings.campaignion_foundation.webform.current_step || 1;
```

Comment:
Although browser support is okish by now, it’s quite the paradigm shift to start using `let` and `const` in Drupal behaviors? I’d rather opt for broader support in this case and stick to `var`…